cookiecutter-xblock/hooks/post_gen_project.py

```diff
@@ -0,0 +1,87 @@
+"""
+Post-generation cookiecutter hook.
+
+* See docs/decisions/0003-layered-cookiecutter.rst
+"""
+import os
+import shutil
```

Comment:
I'm not sure how much you want to try to make changes here, but a lot of this seems like boilerplate that could be consolidated. But maybe that's just an issue with the way we're hacking cookiecutter, and this is actually the cleanest solution.

Reply:
I've almost decided to pull the plug on consolidation of boiler plate into one. Previously, I didn't do that cause I wanted people to be able to create a new repo by calling: "cookiecutter git_url --directory cookeicutter". But now that we've decided to put all tests in parent directory and require edx-lint to be installed for a successfull build, that is no longer really as straight forward as I'd want. So yes, consolidation will probably happen soonish.